spec/unit/bootstrap/definitions_spec.rb

```diff
@@ -0,0 +1,29 @@
+describe FHIR::Definitions do
+  context '.load_extensions'
```

Comment:
Sorry — I wasn't clear! `#load_extensions` isn't tested here. (Here being "my new RSpec tests.") It may be tested elsewhere, but I left the context as a reminder to myself or others that it wasn't directly tested.

Unsure if you guys are covering `get_display` specifically elsewhere, but I came up with a few test scenarios presented in this file. :)